packages/ui/src/assets/icons/generated/index.ts

```diff
@@ -0,0 +1 @@
+export const placeholder = undefined;
```

Comment:
Can we remove this? Any specific reason this file exists? There should be no user created files inside the `generated` folder

Reply:
it can be removed yes, i just wanted to create a placeholder for when there are not any files generated yet so that there wont be any errors when importing ./generated.